genienlp/predict.py

```diff
@@ -223,6 +223,11 @@ def parse_argv(parser):
         action='store_true',
         help='split examples with multiple sentences into individual examples',
     )
+    parser.add_argument(
+        '--translate_only_entities',
```

Comment:
**only** entities as opposed to what? translating full sentences? Can you explain what this does?
We might need to use a more descriptive name instead of `translate_only_entities`.

Reply:
Yeah as opposed to the full sentence. If this argument is provided, the entities are extracted from the sentence and translated individually.  The output will be a dictionary of entities and their translations per sentence.